README.md

```diff
@@ -33,65 +32,198 @@ Only time will tell if making our product roadmap
+  **c)** ***Track progress*** of the task. <br />
+  **d)** ***Confirm*** it was completed to expected standard. <br />
+  **e)** ***Review*** & ***Reflect*** - confirm the task was completed
+  and all the original criteria were met by the assignee.
+  Leave a brief feedback comment with any thoughts.
+  Check the item off as "Done" if appropriate.
+
+For the first step (***Capture***),
+we will build an ultra-simple interface
+to allow people to
+input **`palintext`**
+and save it to a backend as **`text`**.
+For details, please see:
+[`brain-dump.md`](https://github.com/dwyl/product-roadmap/blob/master/checklists.md)
+
+Once the text input is working,
+we will iterate and improve it
+in several interesting and innovative ways
+in response to feedback from people _using_ the MVP. <br />
+We will add additional methods
+to help people "capture" information.
+e.g:
++ Linking to external sources of information.
++ Photo/image uploads
+  + Optical Character Recognition (OCR)
+  + Image/object recognition
++ Voice/audio recording
+  + Speech transcription and indexing
+
+These are complex features
+that have formed the basis for entire companies.
+So in order to make progress with them,
+we will need to _significantly_ leverage
+the Open Source work of many brilliant people.
+But we are getting ahead of ourselves;
+we need to _first_ focus
+on creating "MVP" versions
+of each step in the workflow.
+
+For the _second_ step, ***Categorise***,
+we will need a tagging system that allows people
+to apply textual labels to items.
+The UX for this is still "TBD".
+We will need to sketch it out ahead of sprint planning.
+See: https://github.com/dwyl/product-roadmap/issues/15
+
+
+And finally for the _third_ step, ***Complete***,
+we will build a simple checklist system.
+See: [`checklists.md`](https://github.com/dwyl/product-roadmap/blob/master/checklists.md)
+
+
+
+
+
+### USP: What is _different_ about the @dwyl App?
+
+A valid/relevant **question** is: what makes the @dwyl App unique?
+i.e. What is our
+["unique selling point"](https://en.wikipedia.org/wiki/Unique_selling_proposition)?<br />
+The **answer** is: _initially_ it may _appear_ that our App
+does not have much different
+from existing apps that _partially_ solve the personal effectiveness challenge.
+The _major_ differences will become apparent in due course:
+1. **People Not _Profits_** - We are playing an
+["***infinite game***"](https://github.com/dwyl/start-here/issues/190)
+where our _mission_ is to
+"_Empower people to maximise effectiveness, creativity and happiness_."
+See:
+[What is dwyl's mission?](https://github.com/dwyl/start-here/blob/master/mission.md#what-is-dwyls-mission)
+We will not _rest_ until we have created a system/app
+that is _universally accessible_ to every human being
+that allows _everyone_ a fair opportunity to live their best life.
+We will _cooperate_ with _many_ people/organisations
+to build a comprehensive solution that _anyone_ can use.
```

Comment:
This will require clarification. We don't want people to look at this and think 'it's an app that will try to be all things to all people and therefore is guaranteed to fail'. We need to add a section in here about a targeted approach and first audience. I'm just adding in these comments as I'm reading through to make sure I can come back to this PR and make any changes later 😊

Reply:
Indeed. I'd be _very_ happy to see your (_and other people's) improvements ... 
https://github.com/dwyl/product-roadmap/graphs/contributors 
![ronery](http://giphygifs.s3.amazonaws.com/media/69ZF2NGvIi0Uw/giphy.gif)